db/schema.rb

```diff
@@ -510,6 +510,16 @@
 
   add_index "schedules", ["facility_id"], :name => "i_schedules_facility_id"
 
+  create_table "splits", :force => true do |t|
```

Comment:
I thought usually when you namespaced a model, it would use the namespace in the table name, otherwise you need to explicitly declare `self.table_name =` in the model.

It might be nice to namespace this table.